memo/views.py

```diff
@@ -125,3 +124,9 @@ def post(self, request, pk):
             return Response(status=status.HTTP_204_NO_CONTENT)
         except:
             return Response(status=status.HTTP_400_BAD_REQUEST)
+
+
+class MyLearningTableAPIView(APIView):
+    def get(self, request):
+        my_learning = Deck.objects.get_my_learning_queryset(request.user)
+        return Response(my_learning, status=status.HTTP_200_OK)
```

Comment:
Just to register the change that I mentioned on a comment before =] 
```suggestion
class MyLearningTableAPIView(APIView):
    def get(self, request):
        my_learning = Deck.objects.get_decks_by_owner(request.user)
        serialized_data = MyLearningTableSerializer(my_learning, many=True)
        return Response(data=serialized_data.data, status=status.HTTP_200_OK)
```